client/src/components/common/ImageUploader.tsx

```diff
@@ -0,0 +1,134 @@
+import api from 'src/api';
+import { putImage } from 'src/api/aws';
+import { Close, Image } from 'src/assets/icons';
+import COLORS from 'src/constants/colors';
+import styled from 'styled-components';
+
+interface ImageUpaloderProps {
+  imageUrls: string[];
+  setImageUrls: React.Dispatch<React.SetStateAction<string[]>>;
+}
+
+const MAX_IMAGE = 10;
+const S3_BUCKET_URL =
+  'https://web-fleamarket-08.s3.ap-northeast-2.amazonaws.com/';
+export default function ImageUploader({
+  imageUrls,
+  setImageUrls,
+}: ImageUpaloderProps) {
+  const handleUploadFile = async (e: React.ChangeEvent<HTMLInputElement>) => {
+    if (imageUrls.length === MAX_IMAGE) return;
+    const uploadInfo = await api.getSignedUrl();
+    const target = e.target as HTMLInputElement;
+    if (!target.files) return;
+    await putImage(uploadInfo.data.signedUrl, target.files[0]);
+    setImageUrls((prev: string[]) => [
+      ...prev,
+      `${S3_BUCKET_URL}${uploadInfo.data.fileName}`,
+    ]);
+  };
+
+  const deleteFile = (index: number) => {
+    setImageUrls((prev: string[]) => [
+      ...prev.filter(
+        (fileName: string, fileIndex: number) => index !== fileIndex
+      ),
+    ]);
+  };
+
+  return (
+    <Container>
+      <Box>
+        <UploadBox>
+          <label>
+            <input
+              type="file"
+              onChange={handleUploadFile}
+              accept=".jpg, .jpeg, .png"
+            />
+            <Image></Image>
+            <span>{`${imageUrls.length}/${MAX_IMAGE}`}</span>
+          </label>
+        </UploadBox>
+      </Box>
+      {imageUrls.map((imageUrl, index) => (
+        <Box key={imageUrl}>
+          <CloseWrapper onClick={() => deleteFile(index)}>
+            <Close></Close>
+          </CloseWrapper>
+          <ImageBox>
+            <UploadImage src={imageUrl}></UploadImage>
+            {index === 0 && <PrimaryTag>대표사진</PrimaryTag>}
+          </ImageBox>
+        </Box>
+      ))}
+    </Container>
+  );
+}
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: row;
+`;
+const Box = styled.div`
+  display: block;
+  width: 5rem;
+  height: 5rem;
+  margin: 0.75rem;
+  background-color: white;
+  position: relative;
+`;
+const CloseWrapper = styled.div`
```

Comment:
CloseWrapper는 button이 더 좋을 거 같습니다~!
```suggestion
const CloseWrapper = styled.button`
```

Reply:
네 수정했습니다:)